examples/preview-panel-in-modal/app.tsx

```diff
@@ -0,0 +1,135 @@
+/** @jsx h */
+import { autocomplete, getAlgoliaResults } from '@algolia/autocomplete-js';
+import algoliasearch from 'algoliasearch/lite';
+import { h, render } from 'preact';
+
+import '@algolia/autocomplete-theme-classic';
+
+const searchClient = algoliasearch(
+  'latency',
+  '6be0576ff61c053d5f9a3225e2a90f76'
+);
+
+autocomplete({
+  container: '#autocomplete',
+  detachedMediaQuery: '',
+  defaultActiveItemId: 0,
+  debug: true,
+  getSources() {
+    return [
+      {
+        sourceId: 'hits',
+        getItems({ query }) {
+          return getAlgoliaResults({
+            searchClient,
+            queries: [
+              {
+                indexName: 'instant_search',
+                query,
+                params: {
+                  hitsPerPage: 8,
+                },
+              },
+            ],
+          });
+        },
+        getItemUrl({ item }) {
+          return item.url;
+        },
+        onActive({ item, setContext }) {
+          setContext({ preview: item });
+        },
+        templates: {
+          item({ item, components }) {
+            return (
+              <a className="aa-ItemLink" href={item.url}>
+                <div className="aa-ItemContent">
+                  <div className="aa-ItemIcon">
+                    <img
+                      src={item.image}
+                      alt={item.name}
+                      width="40"
+                      height="40"
+                    />
+                  </div>
+                  <div className="aa-ItemContentBody">
+                    <div className="aa-ItemContentTitle">
+                      <components.Highlight hit={item} attribute="name" />
+                    </div>
+                  </div>
+                </div>
+              </a>
+            );
+          },
+        },
+      },
+      {
+        sourceId: 'suggestions',
+        getItems({ query }) {
+          return getAlgoliaResults({
+            searchClient,
+            queries: [
+              {
+                indexName: 'instantsearch_query_suggestions',
+                query,
+                params: {
+                  hitsPerPage: 4,
+                },
+              },
+            ],
+          });
+        },
+        onSelect({ item, setQuery, setIsOpen, refresh }) {
+          setQuery(`${item.query} `);
+          setIsOpen(true);
+          refresh();
+        },
+        templates: {
+          header({ items, Fragment }) {
+            if (items.length === 0) {
+              return null;
+            }
+
+            return (
+              <Fragment>
+                <span className="aa-SourceHeaderTitle">Suggestions</span>
+                <div className="aa-SourceHeaderLine" />
+              </Fragment>
```

Comment:
How about a similar "Can't find what you're looking for?" text to our docs?

Reply:
Yes! I had this initially and removed it because it looked weird with the then version of the theme. Done in 750f1d88.